src/devices/src/virtio/net/device.rs

```diff
@@ -4,11 +4,12 @@
 // Portions Copyright 2017 The Chromium OS Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the THIRD-PARTY file.
-
+use std::fs::File;
```

Comment:
The rest of the imports seem to be alphabetical order. Not sure if the format checker will catch it.

Reply:
The `cargo fmt --all` actually places it right there.